templates/index.html

```diff
@@ -2,11 +2,11 @@
 
 {% block content %}
 <div class="page-header">
-  <h1>Welcome to SILPA!</h1>
+  <h1>Welcome to Libindic!</h1>
 </div>
 <div>
-  <p class="lead">Swathanthra Indian Language Processing Applications(SILPA) is a web platform to host Free Software language processing applications. It consists of a web framework and a set of applications for processing Indian languages. It is a platform for porting existing and upcoming language processing applications to the web.</p>
-  <p class="lead">Silpa can also be used as a python library or as a webservice from other applications. Silpa is currently under development. If you are interested in contributing contact  <a href="{{url_for('contact')}}">the developers</a></p>
+  <p class="lead">Libindic is a web platform to host Free Software language processing applications. It consists of a web framework and a set of applications for processing Indian languages. It is a platform for porting existing and upcoming language processing applications to the web.</p>
+  <p class="lead">Libindic can also be used as a python library or as a webservice from other applications. Libindic is currently under development. If you are interested in contributing contact  <a href="{{url_for('contact')}}">the developers</a></p>
```

Comment:
@jishnu7 : I had mentioned about writing "Libindic(previously SILPA)" but anivar suggested me not to. Should I add that? 
And why is the travis failing ?